src/collections/src/NICollectionViewModel.h

```diff
@@ -41,7 +41,7 @@
  *
  * @ingroup CollectionViewModels
  */
-@protocol NICollectionViewModeling <NIActionsDataSource, UICollectionViewDataSource>
+@protocol NICollectionViewModeling <NIActionsDataSource, UICollectionViewDataSource, UICollectionViewDataSourcePrefetching>
```

Comment:
I think it's safe to assume that all Nimbus clients should be targeting the iOS 10 SDK or later and that it's reasonable to directly reference the protocol introduced in the iOS 10 SDK 🤔

Reply:
I have concern at first as well, but I think as you mentioned, shouldn't be a big issue now